csl-variables.rnc

```diff
@@ -88,6 +73,11 @@ div {
     | "authority"
     | "call-number"
     | "citation-label"
+    | "collection-title-main"
```

Comment:
It's part of the data structure in the CSL JSON. This would work the same way as for a date. We don't specify `issued-year` as a separate variable. `year` is just a property of the `issued` array.

A title might be supplied as a string or as an array containing main, sub, etc.

Reply:
Ok, have removed the spurios `-main` and `-sub` variables.
But again, how can users supply the main part if the heuristics fail?

Reply:
In the CSL JSON data.

Reply:
Oh, I see that case. Let me think.

Reply:
I mean: in pandoc it won't be a problem. But what about Zotero?

Reply:
Ok, this here does work for issued: I have a date with 2019 in the regular `date` field, and this here in `extra`: `issued: ["year": "2020"]. Result is 2020. Not a pretty solution, but it works.